js/views/pauseScreen.js

```diff
@@ -216,11 +219,19 @@ var PauseScreen = React.createClass({
         controlBarVisible={this.state.controlBarVisible}
       /> : null;
 
+    var containsTitle = this.props.skinConfig.pauseScreen.showTitle && !!title;
+    var containsDescription = this.props.skinConfig.pauseScreen.showDescription && !!description;
+    var containsText = containsTitle || containsDescription;
+
+    if (containsText) {
+      this.props.controller.addBlur();
```

Comment:
Does this need to be executed on each render or would it be better to execute it once on `componentDidMount`?

Reply:
Added a new state ``containsText`` which will control the blur and the fade underlay. Moved the ``addBlur`` call into ``componentDidMount``.